src/IconBase.js

```diff
@@ -25,6 +26,7 @@ IconBase.propTypes = {
   size: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   height: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  opacity: PropTypes.number,
```

Comment:
Q: does it really have to be a number? Because where you're using it, you're providing the `'0.84'` as a string.

Reply:
I just tested this out and it seems that it can just be a number, and that you don't have to make a string out of it.